ouroboros-consensus-shelley/test/Test/ThreadNet/TxGen/Shelley.hs

```diff
@@ -67,8 +70,10 @@ genTx _cfg TickedLedgerState { tickedSlotNo, tickedLedgerState } genEnv =
     mkShelleyTx <$> Gen.genTx
       genEnv
       ledgerEnv
-      (utxoSt, dpState)
+      (utxoSt, dpState) `suchThat` simpleTx
   where
+    simpleTx (STx._body -> txb) =
```

Comment:
Rename to `isSimpleTx` (makes the `suchThat` line more readable) and add a comment saying what we disable and why.